cmd/template/cluster/provider/templates/openstack/machine_deployment.yaml.tmpl

```diff
@@ -0,0 +1,40 @@
+apiVersion: cluster.x-k8s.io/v1alpha4
+kind: MachineDeployment
```

Comment:
Can we use machinepools on Openstack? Machinedeployments are not nice to manage with gitops.

Reply:
We discussed this in Slack.